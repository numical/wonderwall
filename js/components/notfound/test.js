'use strict';
/* eslint-env jasmine */

import React from 'react';
import { Link } from 'react-router';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import ConfigPage from './index.js';

describe('Not Found Page ', () => {
  let page;

  beforeAll(() => {
    page = shallow(<ConfigPage />);
  });

  it('should match previous snapshot', () => {
    expect(shallowToJson(page)).toMatchSnapshot();
  });

  it('should say Page Not Found', () => {
    expect(page.find('.page-title').text()).toEqual('Page Not Found');
  });

  it('should have a Link to home  page', () => {
    expect(page.find(Link).prop('to')).toEqual('/');
  });
});

