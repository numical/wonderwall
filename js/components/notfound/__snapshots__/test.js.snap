exports[`Not Found Page  should match previous snapshot 1`] = `
<div>
  <div
    className="page-title">
    Page Not Found
  </div>
  <Link
    onlyActiveOnIndex={false}
    style={Object {}}
    to="/">
    Back to Wall
  </Link>
</div>
`;
